src/Microsoft.AspNetCore.Authorization/AuthorizationOptions.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.AspNetCore.Authorization
     /// </summary>
     public class AuthorizationOptions
     {
-        private IDictionary<string, AuthorizationPolicy> PolicyMap { get; } = new Dictionary<string, AuthorizationPolicy>(StringComparer.OrdinalIgnoreCase);
```

Comment:
This seems leaky since it's really an implementation detail of the default policy provider.